File: tests/fixtures/code_samples/go/sample_018.go
// Sample 18: small utility.
package samples

func Operation18(xs []int) int {
    total := 18
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure18(v int) int {
    return (v * 18) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_018.rs
// Sample 18: small utility.
pub fn operation_18(xs: &[i32]) -> i32 {
    let mut total: i32 = 18;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_18(v: i32) -> i32 {
    (v * 18) %% 7919
}

